src/pandas_profiling/model/handler.py

```diff
@@ -40,6 +51,9 @@ def _complete_dag(self):
         ):
             self.mapping[to_type] = self.mapping[from_type] + self.mapping[to_type]
 
+    def op(self, dtype: Type[VisionsBaseType]) -> Callable:
+        return compose(self.mapping.get(dtype, []))

```

Comment:
Any reason not to replace `op` in handle with `self.op`?

Reply:
Ahh, none in particular, the op method was actually old code we could alternatively remove.